src/battle_util.c

```diff
@@ -3996,14 +3996,12 @@ static inline uq4_12_t GetSupremeOverlordModifier(u32 battler)
     return UQ_4_12(1.0) + (UQ_4_12(0.1) * gBattleStruct->supremeOverlordCounter[battler]);
 }
 
-static inline bool32 HadMoreThanHalfHpNowHasLess(u32 battler)
+static inline bool32 HadMoreThanHalfHpNowDoesnt(u32 battler)
```

Comment:
What precisely is the reasoning behind this rename? I don't see how it makes it more clear than before

Reply:
Because it can be equal to half also.

Reply:
That's fair, but how does the new name clarify that? It still indicates that it no longer has more than half HP (thus excluding equal to half). I'd personally go for something like `HadEqualToOrMoreThanHalfHpNowHasLess` but that is probably overly verbose

Reply:
Before HP > Half
After HP <= Half
`HadMoreThanHalfHpNowDoesnt` is the least verbose but accurate name I could think of.
`HadMoreThanHalfHpNowHasHalfOrLess` is up next.

Reply:
Ohh, I misunderstood. I'm a fan of `HadMoreThanHalfHpNowDoesnt` in that case!